_includes/analytics.js

```diff
@@ -0,0 +1,12 @@
+// Analytics script to dynamically add the googletagmanager script
```

Comment:
Moved it into this script from the HTML so I can add it dynamically based on cookie consent.